pkgs/by-name/mi/miniflux/package.nix

```
@@ -24,6 +24,10 @@ buildGoModule rec {
   postInstall = ''
     mv $out/bin/miniflux.app $out/bin/miniflux
     installManPage miniflux.1
+
+    # Upstream vendors a Systemd service.
```

```suggestion
    # Upstream provides a Systemd service.
```
vendors is the wing term here. Same with the commit and pr title

---

How is it the wrong term?

---

I would like to know that myself. Upstream *does vendor* stuff - this change is not necessary.

---

Vendoring usually means taking something from external and moving it into the repo/bundle. In this case, there is no upstream external repository and they've written this unit file from scratch for their service.

---

> vendors is the wing term here. Same with the commit and pr title

I'm sorry but this one of the most pedantic review i have witnessed from you during my time in nixpkgs